Resources/Locale/en-US/traitor/uplink/telecrystal-component.ftl

```diff
@@ -1 +1 @@
-telecrystal-component-sucs-inserted = You slot {$source} into the {$target}.
\ No newline at end of file
+telecrystal-component-sucs-inserted = You slot the telecrystals into the uplink
```

Comment:
this is the incorrect way to fix this, pass in the args properly when this is called and then do something like this

```suggestion
telecrystal-component-sucs-inserted = You slot {THE($source)} into {THE($target)}.
```